Tasks/PublishBuildArtifacts/task.json

```diff
@@ -65,15 +57,9 @@
     }
   ],
   "instanceNameFormat": "Publish Artifact: $(ArtifactName)",
-  "execution": {
-    "PowerShell": {
-      "target": "$(currentDirectory)\\PublishBuildArtifacts.ps1",
-      "argumentFormat": "",
-      "workingDirectory": "$(currentDirectory)",
-      "platforms" : ["windows"]
-    },
+  "execution": {    
     "Node": {
-      "target": "publishBuildArtifacts.js",
+      "target": "PublishBuildArtifacts.js",
```

Comment:
Why did you change this?  it's actually publishBuildArtifacts and *nix is case sensitive